aspnetcore/host-and-deploy/azure-apps/troubleshoot.md

```diff
@@ -65,11 +65,48 @@ Many startup errors don't produce useful information in the Application Event Lo
 
 1. Open **Advanced Tools** in the **Development Tools** area. Select the **Go&rarr;** button. The Kudu console opens in a new browser tab or window.
 1. Using the navigation bar at the top of the page, open **Debug console** and select **CMD**.
-1. Open the folders to the path **site** > **wwwroot**.
-1. In the console, run the app by executing the app's assembly.
-   * If the app is a [framework-dependent deployment](/dotnet/core/deploying/#framework-dependent-deployments-fdd), run the app's assembly with *dotnet.exe*. In the following command, substitute the name of the app's assembly for `<assembly_name>`: `dotnet .\<assembly_name>.dll`
-   * If the app is a [self-contained deployment](/dotnet/core/deploying/#self-contained-deployments-scd), run the app's executable. In the following command, substitute the name of the app's assembly for `<assembly_name>`: `<assembly_name>.exe`
-1. The console output from the app, showing any errors, is piped to the Kudu console.
+
+#### Test a 32-bit (x86) app
+
+##### Current release
+
+1. Execute `cd d:\home\site\wwwroot`.
+1. Run the app:
+   * If the app is a [framework-dependent deployment](/dotnet/core/deploying/#framework-dependent-deployments-fdd), run the app with `dotnet .\{ASSEMBLY NAME}.dll`.
+   * If the app is a [self-contained deployment](/dotnet/core/deploying/#self-contained-deployments-scd), run the app with `{ASSEMBLY NAME}.exe`.
+   
+The console output from the app, showing any errors, is piped to the Kudu console.
+   
+##### Framework-depdendent deployment running on a preview release
+
+*Requires installation of the the ASP.NET Core {VERSION} (x86) Runtime site extension.*
+
+1. Execute `cd D:\home\SiteExtensions\AspNetCoreRuntime.{X.Y}.x32`, where `{X.Y}` is the runtime version.
+1. Run the app: `dotnet \home\site\wwwroot\{ASSEMBLY NAME}.dll`.
+
+The console output from the app, showing any errors, is piped to the Kudu console.
+
+#### Test a 64-bit (x64) app
+
+##### Current release
+
+* If the app is a 64-bit (x64) [framework-dependent deployment](/dotnet/core/deploying/#framework-dependent-deployments-fdd):
+  1. Execute `cd D:\Program Files\dotnet`.
```

Comment:
This is an advanced tutorial. Execute is not approved on pollicheck. I don't think it MT well. It adds nothing and subtracts much.

cc @tdykstra